src/macros.jl

```diff
@@ -5,6 +5,9 @@
 
 using Base.Meta
 
+issum(s::Symbol) = (s == :sum) || (s == :∑) || (s == :Σ)
```

Comment:
I was going to suggest casting the variable to lowercase so that you didn't have to handle comparing to both uppercase and lowercase, but I was disappointed to discover that Julia does not handle case transformations for Greek. 

```julia
lowercase("∑") == "∑" # False
lowercase("E") == "e" # True
```